apps/frontend/src/app/3_organisms/RuneBridgeDialog/components/SendFlow/components/AmountForm.tsx

```diff
@@ -204,7 +207,7 @@ export const AmountForm: React.FC = () => {
                   ? `${limits.flatFeeTokens} ${selectedToken.symbol}`
                   : '',
                 limits.flatFeeBaseCurrency
-                  ? `${limits.flatFeeBaseCurrency} ${BITCOIN}`
+                  ? `${limits.flatFeeBaseCurrency} ${baseCurrency}`
```

Comment:
Will we have the same service fee for RSK and BoB? Because it's always 0.4% + 0.0003 [ETH/BTC] but 0.0003 BTC is much more in fiat than 0.0003 ETH.

Reply:
This fee is taken from the contract so it can be changed without updating the UI code. It probably won't be the same as with BTC.

Reply:
Ok, I just wanted to make sure.